ethereum/core/src/test/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessorIntegrationTest.java

```diff
@@ -0,0 +1,758 @@
+  private static final String ACCOUNT_3 = "0x0000000000000000000000000000000000000003";
+  private static final String ACCOUNT_4 = "0x0000000000000000000000000000000000000004";
+  private static final String ACCOUNT_5 = "0x0000000000000000000000000000000000000005";
+  private static final String ACCOUNT_6 = "0x0000000000000000000000000000000000000006";
+  private static final String CONTRACT_ADDRESS = "0x00000000000000000000000000000000000fffff";
+
+  private static final KeyPair keyPair1 =
+      generateKeyPair("c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3");
+
+  private static final KeyPair keyPair2 =
+      generateKeyPair("fc5141e75bf622179f8eedada7fab3e2e6b3e3da8eb9df4f46d84df22df7430e");
+
+  private WorldStateArchive worldStateArchive;
+  private MainnetParallelBlockProcessor parallelBlockProcessor;
+  private BlockProcessor blockProcessor;
+  private DefaultBlockchain blockchain;
+  private Address coinbase;
+
+  @BeforeEach
+  public void setUp() {
+    final ExecutionContextTestFixture contextTestFixture =
+        ExecutionContextTestFixture.builder(
+                GenesisConfigFile.fromResource(
+                    "/org/hyperledger/besu/ethereum/mainnet/genesis-bp-it.json"))
+            .dataStorageFormat(DataStorageFormat.BONSAI)
+            .build();
+    final ProtocolSchedule protocolSchedule = contextTestFixture.getProtocolSchedule();
+    final BlockHeader blockHeader = new BlockHeaderTestFixture().number(0L).buildHeader();
+    final MainnetTransactionProcessor transactionProcessor =
+        protocolSchedule.getByBlockHeader(blockHeader).getTransactionProcessor();
+    coinbase = blockHeader.getCoinbase();
+    worldStateArchive = contextTestFixture.getStateArchive();
+    blockchain = (DefaultBlockchain) contextTestFixture.getBlockchain();
+    blockProcessor =
+        new MainnetBlockProcessor(
+            transactionProcessor,
+            protocolSchedule
+                .getByBlockHeader(new BlockHeaderTestFixture().number(0L).buildHeader())
+                .getTransactionReceiptFactory(),
+            Wei.of(2_000_000_000_000_000L),
+            BlockHeader::getCoinbase,
+            false,
+            protocolSchedule);
+    parallelBlockProcessor =
+        new MainnetParallelBlockProcessor(
+            transactionProcessor,
+            protocolSchedule
+                .getByBlockHeader(new BlockHeaderTestFixture().number(0L).buildHeader())
+                .getTransactionReceiptFactory(),
+            Wei.of(2_000_000_000_000_000L),
+            BlockHeader::getCoinbase,
+            false,
+            protocolSchedule,
+            new NoOpMetricsSystem());
+  }
+
+  @Test
+  void testSequentialBlockProcessingwithTransfers() {
+    processSimpleTransfers(blockProcessor);
+  }
+
+  @Test
+  void testParallelBlockProcessingWithTransfers() {
+    processSimpleTransfers(parallelBlockProcessor);
+  }
+
+  @Test
+  void testSequentiaConfiltedTransfers1() {
```

Comment:
What is the difference between Transfers1 and Transfers2? Could this be more descriptive?

Reply:
The difference is that Transfers1 are conflicting because of the same sender, and Transfers2 are confiling because the receiver of the first transaction is the sender of the second transaction.
I renamed the tests.